test/test_nn.py

```diff
@@ -60,9 +60,15 @@
 
 AMPERE_OR_ROCM = TEST_WITH_ROCM or torch.cuda.is_tf32_supported()
 
+
+TEST_GPU = torch.cuda.is_available() or torch.xpu.is_available()
```

Comment:
why does not use `torch.accelerator.is_available`?

Reply:
Updated, thanks!